datapackage_pipelines_mojp/bagnowka/processors/convert.py

```diff
@@ -15,25 +15,41 @@ def _get_schema(self):
     def _filter_resource(self, resource_descriptor, resource_data):
         for cm_row in resource_data:
             dbs_row = self._cm_row_to_dbs_row(cm_row)
+            logging.info("hello CONVERT")
             yield dbs_row
 
     def _cm_row_to_dbs_row(self, cm_row):
```

Comment:
the `cm` in cm_row is meant to signify `clearmash`
better to rename it to `bagnowka_row` or something like that to prevent confusion